### DISCUSSION TITLE
pscale login should allow --org

### DISCUSSION THREAD

**Discussion by mohsen1** [planetscale/discussion#280] [Bug Reports] [answered]
So I can login to a different org

**Comment by fooforge** [comment, marked as answer]
Hey @mohsen1,

I'm not sure I understand your request. The `--org` flag allows you to switch between the organizations you're a member of. It's available on commands where it makes sense, e.g. `pscale branch` or `pscale database`. The default organization can be changed by editing `~/.config/planetscale/pscale.yml`.

Were you perhaps trying to say that `pscale login` should support multiple logged in users instead?
